test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/SetTest.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.jsinterop
+
+import org.junit.Assert._
+import org.junit.Assume.assumeTrue
+import org.junit.{BeforeClass, Test}
+import org.scalajs.testsuite.utils.Platform._
+
+import scala.scalajs.js
+
+object SetTest {
+  @BeforeClass
+  def assumeRuntimeSupportsSet(): Unit = {
+    assumeTrue("Assume Symbol and Set exists in Global",
+        jsSymbols && js.typeOf(js.constructorOf[js.Set[_]]) != "undefined")
```

Comment:
You didn't answer my question in the previous review: why testing for `Symbol` here? I don't think it's necessary.